README.rst

```diff
@@ -123,13 +123,18 @@ waste CPU instructions. Either prepend ``assert`` or remove it.
 **B016**: Cannot raise a literal. Did you intend to return it or raise
 an Exception?
 
-**B017**: ``self.assertRaises(Exception):`` should be considered evil. It can lead 
-to your test passing even if the code being tested is never executed due to a typo. 
-Either assert for a more specific exception (builtin or custom), use 
+**B017**: ``self.assertRaises(Exception):`` should be considered evil. It can lead
+to your test passing even if the code being tested is never executed due to a typo.
+Either assert for a more specific exception (builtin or custom), use
 ``assertRaisesRegex``, or use the context manager form of assertRaises
 (``with self.assertRaises(Exception) as ex:``) with an assertion against the
 data available in ``ex``.
 
+**B018**: Within an ``except`` clause, raise exceptions with ``raise ... from err``
+or ``raise ... from None`` to distinguish them from errors in exception handling.
+See [the exception chaining tutorial](https://docs.python.org/3/tutorial/errors.html#exception-chaining)
+for details.
+
```

Comment:
Do we need to care to make this only care if we're >= 3.9?
```suggestion
**B018**: Within an ``except`` clause, raise exceptions with ``raise ... from err``
or ``raise ... from None`` to distinguish them from errors in exception handling (i.e. prefer exception chaining).
See [the exception chaining tutorial](https://docs.python.org/3/tutorial/errors.html#exception-chaining)
for details.

```

Reply:
Exception chaining was added in Python 3.0!  It's just not in widespread use yet, due to the syntatic incompatibility with Python 2 😅

Reply:
O - I went to docs here: https://docs.python.org/3.9/tutorial/errors.html#exception-chaining
- Set it to 3.8 and this section disappears ...

Reply:
Wow, I guess it's not been well documented... [the reference docs](https://docs.python.org/3.8/library/exceptions.html) still don't have a heading for it, though `ctrl-f chain` does find some mentions.